### DISCUSSION TITLE
Support for MI-700  Unterstützung für

### DISCUSSION THREAD

**Discussion by Ricsm63** [tbnobody/OpenDTU#90] [Ideas]
I think HM-700 & MI-700 are the same, one model is for America and the other for Europe.
Is it possible to add support for the MI-700?
The serial numbers for MI-700 are 1042********
Ich denke, HM-700 und MI-700 sind gleich, ein Modell ist für Amerika und das andere für Europa.
Ist es möglich, Unterstützung für das MI-700 hinzuzufügen?
Die Seriennummern für MI-700 sind 1042********
![1](https://user-images.githubusercontent.com/111537625/185526751-d5fdf88c-121a-4b81-816d-e7e6a0ecaad1.png)
![2](https://user-images.githubusercontent.com/111537625/185526753-aa65775d-b816-423d-8383-acea209c8e14.png)
![3](https://user-images.githubusercontent.com/111537625/185526755-a8a75993-ce74-4de9-9c2f-458632beb81a.png)
![4](https://user-images.githubusercontent.com/111537625/185526757-60917de6-bcc6-480a-89e9-e53709b757e7.png)

**Comment by tbnobody** [comment]
> I think HM-700 & MI-700 are the same, one model is for America and the other for Europe.

Nope, they are using a completly different telegram structure. Please see the mikrocontroller.net thread mentioned in the README.md. There are already some very early implementations of the MI devices. (MI devices using a older and earlier communication protocol)
